### PR TITLE
Adding support for certificates

### PR DESCRIPTION
**Pull request by ddavila0**
With this, you can use certificates instead of username/password to send data

### CODE REVIEW

File: src/python/WMCore/Services/StompAMQ/StompAMQ.py [hunk 1]
```diff
@@ -61,19 +61,24 @@ class StompAMQ(object):
...
+                 host_and_ports=None, logger=None, cert=None, key=None, use_ssl=False):
         self._username = username
         self._password = password
         self._producer = producer
         self._topic = topic
         self._host_and_ports = host_and_ports or [('agileinf-mb.cern.ch', 61213)]
         self.logger = logger if logger else logging.getLogger()
+        self._cert=cert
+        self._key=key
+        self._use_ssl=use_ssl
```
Comment (vkuznet) [inline src/python/WMCore/Services/StompAMQ/StompAMQ.py]: Can we drop `use_ssl` completely and do instead
```
self._use_ssl = True if key and cert else False
```

File: src/python/WMCore/Services/StompAMQ/StompAMQ.py [hunk 2]
```diff
@@ -90,10 +95,19 @@ def send(self, data):
             return data
 
         conn = stomp.Connection(host_and_ports=self._host_and_ports)
+
+        if(self._use_ssl):
```
Comment (amaltaro) [inline src/python/WMCore/Services/StompAMQ/StompAMQ.py]: Update it to `if self._use_ssl:`, same below

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
@ddavila0 as we discussed over gtalk, please
1. test these changes 
2. squash those commits

**Pull request closed without merging**